src/cmds/hardware/usb_whitelist.c

```diff
@@ -258,8 +259,14 @@ int main(int argc, char *argv[]) {
 		return 0;
 	}
 
-	strncpy(usb_wl_dev_path, "/dev/", USB_WL_DEV_PATH_LEN);
-	strncat(usb_wl_dev_path, USB_WHITELIST_DEV_NAME, USB_WL_DEV_PATH_LEN);
+	strncpy(usb_wl_dev_path,
```

Comment:
I think we can just use `strcpy` instead, and then do `strncat` to the result which appends null byte. There is no need to use `strncpy` here since we explicitly assume the size of `"/dev/"` is lower than `sizeof(usb_wl_dev_path)`.

Reply:
I have no idea why GitHub shows this commit, it's already merged to master.

Reply:
Ah, ok